plugins/vppv2/natplugin/vppcalls/nat_vppcalls.go

```diff
@@ -36,6 +38,10 @@ const (
 	NoInterface = ^uint32(0)
 	// Maximal length of tag
 	maxTagLen = 64
+	// label added to tags of DNAT mappings with external IP from the pool.
+	extIPFromPoolLabel = "POOL" // keep it short, there is a limit on the maximum tag length
```

Comment:
So it seems we can't work without adding some metadata to the tag. Could you please explain to me what this represents? (I have limited experience with NAT)

Perhaps it would be better keep only `P` instead of `POOL` to save 3 characters.

Reply:
This "ExternalIpFromPool" thing is actually a temporary workaround for bug in the VPP NAT plugin - Matus will soon provide a fix. The plan is to revert this before we merge dev into master, but for now I needed it to unblock Contiv refactoring.
Otherwise we really do not need any extra labels to add into the tags.